src/constants/Features.js

```
@@ -0,0 +1,37 @@
+import { CircleDollarSign, Infinity, Scale } from "lucide-react";
```

Biome (1.9.4)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)